pages/common/virt-sysprep.md

```diff
@@ -0,0 +1,20 @@
+# virt-sysprep
+
+> Reset, unconfigure, or customize a virtual machine image.
+> More information: <https://manned.org/virt-sysprep>.
+
+- List all supported operations (enabled operations are indicated with asterisks):
+
+`virt-sysprep --list-operations`
+
+- Run all enabled operations but don't apply the changes:
+
+`virt-sysprep --domain {{vm_name}} --dry-run`
+
+- Run only the specified operations:
+
+`virt-sysprep --domain {{vm_name}} --operations {{operation1,operation2,...}}`
+
+- Generate a new `/etc/machine-id` and enable customizations to be able to change the host name to avoid network conflicts:
```

Comment:
```suggestion
- Generate a new `/etc/machine-id` file and enable customizations to be able to change the host name to avoid network conflicts:
```
Also, I'm not swallowing "to be able to change the host name".  Can you elaborate that for me?

Reply:
If --enable customizations isn't specified, --hostname throws an error. I could say "enable customizations to enable setting the host name".